pom.xml

```diff
@@ -89,6 +89,21 @@
             </ignores>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <executions>
+            <execution>
+              <goals>
+                <goal>compile</goal>
+              </goals>
+            </execution>
+          </executions>
```

Comment:
No need to override this aspect of the superpom.

```suggestion
```

Reply:
Thanks for the suggestion. It should be obvious that I have very little knowledge, let alone understanding, of Maven.